crossbeam-utils/src/thread.rs

```diff
@@ -522,6 +522,35 @@ impl<'scope, T> ScopedJoinHandle<'scope, T> {
     }
 }
 
+#[cfg(unix)]
```

Comment:
Is it possible to use cfg-if?  IMO, it would look better.

Reply:
done